src/libraries/Microsoft.Extensions.Logging.EventSource/src/LoggingEventSource.cs

```diff
@@ -451,7 +451,7 @@ private static bool TryParseLevel(LogLevel defaultLevel, string levelString, out
                     {
                         return false;
                     }
-                    if (!(LogLevel.Trace <= (LogLevel)level && (LogLevel)level <= LogLevel.None))
+                    if ((LogLevel)level is not (>= LogLevel.Trace and <= LogLevel.None))
```

Comment:
I don't think this is more readable than the current code. Can you revert this change?

Reply:
Yeah, both are not really readable, but VS suggested it, and I thought there might be some performance improvement. Reverted.